src/documents/ComponentGrades.md

```diff
@@ -17,7 +17,7 @@ fluid.defaults("examples.myGrade", {
 
 Each such grade can be built on to derive further grades/components. This derivation occurs by mentioning the name of the original grade (e.g. `examples.myGrade`) within the **gradeNames** section of the derived component.
 
-Most grades you will deal with are *** component grades *** derived from `fluid.component`. However, for some purposes you may also deal with [*** function grades ***](FunctionGrades.md) which are derived from `fluid.function`.
+Most grades you will deal with are ***component grades*** derived from `fluid.component`. However, for some purposes you may also deal with [*** function grades ***](FunctionGrades.md) which are derived from `fluid.function`.
```

Comment:
I realise that you didn't introduce the `"***"` markup, only fixed it, but I think it's probably not what we want here. The text of this paragraph is without any emphasis and the 3 asterisks are rendered as `<em><strong>`. I think we would want only one of those. A single or double asterisks, not both.

It looks like `"**"` (`<strong>`) is what we have on other parts of the page for similar usage.

https://docs.fluidproject.org/infusion/development/ComponentGrades.html

There's also another `***<space>...<spcace>***` on this line: `[*** function grades ***]`.

Reply:
We use lots of degrees of emphasis, there were some with triple asterisks and in CAPS as well, but I agree, toning it down seems reasonable.